modules/node_modules/@ncigdc/dux/cart.js

```diff
@@ -92,6 +92,29 @@ function toggleFilesInCart(incomingFile) {
   };
 }
 
+function removeFilesFromCart(files) {
+  return (dispatch, getState) => {
+    const filesToRemove = _.isArray(files) ? files : [files];
+    const existingFiles = getState().cart.files;
+    const nextFiles = _.differenceBy(existingFiles, filesToRemove, 'file_id');
+    const filesRemoved = existingFiles.length - nextFiles.length;
+    dispatch(notify(getNotificationComponent('remove', `remove/${filesToRemove.file_name}`, {
```

Comment:
Nice catch

I copied and pasted that part from [toggleFilesInCart](https://github.com/NCI-GDC/portal-ui/blob/baae998a578fba75704da6ddec410dda5c9a9793/modules/node_modules/%40ncigdc/dux/cart.js#L66-L77). How is it handled there?